CONTRIBUTING.md

```diff
@@ -674,7 +674,7 @@ ccache -M 25Gi
 ```
 
 To check this is working, do two clean builds of pytorch in a row. The second
-build should be substantially and noticeably faster than the first build.
+build should be substantially and noticeably faster than the first build. If this doesn't seem to be the case, check that each of the symlinks above actually link to your `ccache`. For example, running `readlink -e $(which g++)` on your Linux terminal should return `~/ccache/bin/ccache` if you installed `ccache` from source.
```

Comment:
I actually get this output from running that command:
```
/data/users/sestep/miniconda3/envs/pytorch/bin/ccache
```
Does this just mean I didn't actually install it from source? Should a sentence about this be added to these docs?

Reply:
This means you installed it through conda, which is valid. I wasn't sure if I should go through all the combinations of platforms and how people installed ccache, which is why I used "For example". Do you think this adds more confusion?

Reply:
I would say that the page should show what that command would return for at least the ways to install `ccache` that are already included in `CONTRIBUTING.md`:

- from source (what you currently have in this PR)
- from `conda` (what I apparently did, shown above)
- from `apt` (although this one might be slightly less convenient for us to test locally)

Reply:
hmm. IMO, as pytorch repo, it is not our responsibility to cover all possible ways to install ccache in different platform. - maybe an external link?

Reply:
@walterddr sure maybe, but my point is that the `CONTRIBUTING.md` file already _does_ detail three different ways to install ccache, so we could cover at least those three different ways in this PR (or remove parts of the existing text if we don't actually want to cover them)

Reply:
Yeah currently it seems a bit confusing with all those 3 methods. it might be a good idea to 
1. simplify it with 1 "recommended" way (conda maybe? it is easiest). then describe the other 2 as backup if fails 
2. describe exactly in what circumstances, should one use each of those methods.

I am open with both. Or if we want to keep the current structure, it's also fine with Jane's addition to the verification method